spec/spec.md

```diff
@@ -185,7 +185,9 @@ A topic ****MUST**** be a [[ref:JSON Schema]] document that can be applied to an
 [[ref:Trust Establishment Documents]] may fit into any number of embed targets – verifiable wrappers or embedded proofing formats - to enable data integrity. This specification takes no position on which means are suitable to provide integrity of [[ref:Trust Establishment Documents]] or [[ref:Topics]], however provide a number of examples for convenience.
```

Comment:
```suggestion
[[ref:Trust Establishment Documents]] may fit into any number of embed targets – verifiable wrappers or embedded proofing formats - to enable data integrity. This specification takes no position on which means are suitable to provide integrity of [[ref:Trust Establishment Documents]] or [[ref:Topics]]; the examples it provides should be taken as illustrative rather than normative in regards to those decisions at higher layers.
```